### PR TITLE
[DEVHUB-1378] & [DEVHUB-1379]: Content Page - Integrate Content & Routing Functionality/User Actions

### PR DESCRIPTION
**Pull request by justinmckenzie**
Scope
- Integrate Course page with Strapi Content
- Implement Social Sharing and Copying Course page to Clipboard
- Adjust Course Page styling and layout
- Remove `pre-requisites` in `<CourseAside />`
- Remove remaining mock data

### CODE REVIEW

File: src/components/course-aside/course-aside.tsx [hunk 1]
```diff
@@ -6,25 +7,43 @@ import CourseAsideProps from './types';
...
-      }}
-      data-testid="course-aside"
-    >
+    <aside sx={{ ...wrapperStyles }} data-testid="course-aside">
       <section sx={styles.CourseAsideSection}>
         <div sx={styles.CourseAsideWidget}>
           <h3 sx={styles.CourseAsideLabel}>Length</h3>
-          <span sx={styles.CourseAsideDetail}>{length}</span>
+          <span sx={styles.CourseAsideDetail}>
+            {duration} Hour-long lectures
```
Comment (mcolella14) [inline src/components/course-aside/course-aside.tsx:45]: I think this should just be `{duration} lectures`. I remember talking about it with someone and these lectures vary in length.

**Review by mcolella14 — APPROVED** [review]
LGTM! Just the thing about the lectures duration.

File: src/components/course-aside/course-aside.tsx (hunk 1 above)
Comment (mcolella14) [reply src/components/course-aside/course-aside.tsx:45]: Wait actually it should just be `{duration} hours` (or `{duration} hour`), sorry got that mixed up.

Comment (justinmckenzie) [reply src/components/course-aside/course-aside.tsx:45]: updating

**Pull request merged**